packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -71,9 +71,11 @@ class RunCommand extends RunCommandBase {
 
     // Option to write the pid to a file.
     argParser.addOption('pid-file',
-                        help: 'Specify a file to write the process id to.'
-                              'You can send SIGUSR1 to trigger a hot reload '
-                              'and SIGUSR2 to trigger a full restart.');
+                        help: """
+                              Specify a file to write the process id to.
+                              You can send SIGUSR1 to trigger a hot reload
+                              and SIGUSR2 to trigger a full restart.
+                              """);
 
```

Comment:
looks like this should have been fixed by using `'` string literals after all, because as written it is indenting the string.